.github/workflows/sonarcloud.yml

```
@@ -111,76 +112,94 @@ jobs:
-              if [ -z "$SOURCES" ]; then
-                SOURCES="project/backend/$service_name/src/main/java"
-              else
-                SOURCES="$SOURCES,project/backend/$service_name/src/main/java"
+            # Collect all the source directories without wildcards
+            SOURCES=""
+            TESTS=""
+            BINARIES=""
+            TEST_BINARIES=""
+            JACOCO_REPORTS=""
+            LIBRARIES=""
+  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)